include/rocksdb/attribute_groups.h

```diff
@@ -77,6 +77,25 @@ inline void PinnableAttributeGroup::Reset() {
 // A collection of Pinnable Attribute Groups.
 using PinnableAttributeGroups = std::vector<PinnableAttributeGroup>;
 
+// Used in Iterator Path. Uses pointers to the columns to avoid having to copy
+// all WideColumns objs during iteration.
+class IterableAttributeGroup {
```

Comment:
`IterableAttributeGroup` suggests to me an attribute group that can be iterated over; how about `IteratorAttributeGroup` ?